jobHandler/run.py

```
@@ -0,0 +1,128 @@
+
+        3. Client runs job using
+        {
+            "code": 2,
+            "client": True,
+            "type": "bubbleSort",
+            "data": [13,6,3,12,3]
+        }
+    '''
+    async for message in websocket:
+        message = json.loads(message)
+        asp_uid = message["asp_uid"]
```

Ruff (0.8.2)

Local variable `asp_uid` is assigned to but never used

Remove assignment to unused variable `asp_uid`

(F841)